cmd/ps.go

```diff
@@ -120,10 +121,16 @@ func printProcesses(appID string, processes []api.Pods) {
 
 	fmt.Printf("=== %s Processes\n", appID)
 
-	for psType, procs := range psMap {
+	var psTypes []string
```

Comment:
Same comment here as [above](https://github.com/deis/workflow-cli/pull/144/files#r72125711). Introducing a `Processes` object into the SDK with the applicable sorting functions required would be the optimal approach, IMO.